samples/observability.js

```diff
@@ -0,0 +1,284 @@
+/*!
+ * Copyright 2024 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//   title: Observability (Tracing) with OpenTelemetry
+//   usage: node observability.js trace <INSTANCE> <DATABASE> <PROJECT-ID>
+
+'use strict';
+
+// Setup OpenTelemetry and the trace exporter.
+// [START spanner_trace_and_export_spans]
+const {Resource} = require('@opentelemetry/resources');
+const {NodeSDK} = require('@opentelemetry/sdk-node');
+const {
+  NodeTracerProvider,
+  TraceIdRatioBasedSampler,
+} = require('@opentelemetry/sdk-trace-node');
```

Comment:
Resolve all github comments.

Reply:
`ConsoleSpanExporter` is not used. `SimpleSpanProcessor` is not used. 

No need to add all possible things. Need to add what we want to recommend in sample.

Reply:
Done, thanks!